### ISSUE TITLE
Typo on line 94 of PostSaveHierarchical.php

### ISSUE THREAD

**Issue by holkki** [publishpress/PublishPress-Permissions#1084]
## Expected Behavior

This function fires when a post is saved and does something, I'm not exactly sure what.

## Current Behavior

This line sometimes causes a fatal error when saving a post.

```
[26-Apr-2024 07:34:21] "NOTICE: PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function PublishPress\Permissions\Collab\defind() in /data/wordpress/wp-content/plugins/press-permit-core/modules/presspermit-collaboration/classes/Permissions/Collab/PostSaveHierarchical.php:94"
[26-Apr-2024 07:34:21] "Stack trace:"
[26-Apr-2024 07:34:21] "#0 /data/wordpress/wp-content/plugins/press-permit-core/modules/presspermit-collaboration/classes/Permissions/Collab/AdminFilters.php(309): PublishPress\Permissions\Collab\PostSaveHierarchical::fltPageParent()"
[26-Apr-2024 07:34:21] "#1 /data/wordpress/wp-includes/class-wp-hook.php(324): PublishPress\Permissions\Collab\AdminFilters->fltPageParent()"
[26-Apr-2024 07:34:21] "#2 /data/wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters()"
[26-Apr-2024 07:34:21] "#3 /data/wordpress/wp-includes/post.php(2926): apply_filters()"
[26-Apr-2024 07:34:21] "#4 /data/wordpress/wp-includes/post.php(2823): sanitize_post_field()"
[26-Apr-2024 07:34:21] "#5 /data/wordpress/wp-includes/post.php(4184): sanitize_post()"
[26-Apr-2024 07:34:21] "#6 /data/wordpress/wp-includes/post.php(4930): wp_insert_post()"
[26-Apr-2024 07:34:21] "#7 /data/wordpress/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller..." 
```
  
## Possible Solution

This probably should be "defined"? 

https://github.com/publishpress/PublishPress-Permissions/blob/dcb5ca6fdf91954b7f360ef7efdec3631b843270/modules/presspermit-collaboration/classes/Permissions/Collab/PostSaveHierarchical.php#L94

`  if (defind('PRESSPERMIT_OWN_DESCENDENT_CHECK')) {`

## Steps to Reproduce (for bugs)

1. Edit post, press save? Unsure, just checked error logs and saw this.

## Context

A customer noticed that their users cannot publish new posts.

## Your Environment

* Version used: 4.0.21
* Browser Name and version: Firefox 125.0.2
* Operating System and version (desktop or mobile): Windows 10
* Link to your project:

**Comment by stevejburge** [comment]
Thanks @holkki

**Comment by lscwebmaster** [comment]
I can confirm this - last nights plugin update broke publishing on our site! This one is critical to be fixed ASAP!

**Comment by stevejburge** [comment]
@lscwebmaster we're looking at this today.

Just a quick note: Github isn't somewhere we provide urgent support. If this is urgent and impacting your site, you can use the support available for Pro members, or rollback to an earlier version of the plugin.

**Comment by lscwebmaster** [comment]
Not sure how the typo made it's way into the production version, not happy! I would say it is urgent and affecting everyone who uses your plugin that was quick to update it with auto-update features in Wordpress. I already had to rollback to yesterday's plugin version and am excluding it from auto-update until you release a fix for this one.

**Comment by lscwebmaster** [comment]
Otherwise, it's a great plugin and has been working wonderful for us, so am happy with that.

**Comment by piholgitub** [comment]
Hello, just to understand the process:
the bug is fixed now and when will the fix released ? And with a new release ?
Thanks and regards
Guido

**Comment by agapetry** [comment]
@piholgitub The fix is released in version 4.0.22

**Comment by piholgitub** [comment]
Thank you
Guido

Il giorno 26 apr 2024, alle ore 20:36, Kevin Behrens ***@***.***> ha scritto:


@piholgitub <https://github.com/piholgitub> The fix is released in version 4.0.22
